huggingface/tensorflow/inference/docker/2.6/py3/cu112/Dockerfile.gpu

```diff
@@ -42,6 +42,8 @@ RUN apt-get update \
     unzip \
     git \
     libnuma1 \
+    libsndfile1-dev \
+    ffmpeg \
```

Comment:
These seem to be installed only on the inference dockerfiles, and not the training dockerfiles. Is this intentional?

We should have some kind of validation checks to ensure that hugging face DLCs for TF>=2.6 and PT>=1.10 have these libs installed, as well as tests to validate that the audio and vision transformers packages are installed.

Example: https://github.com/aws/deep-learning-containers/blob/master/test/dlc_tests/sanity/test_utility_installation.py

Reply:
Yes they are used in the [transformers.pipeline](https://github.com/huggingface/transformers/blob/b320d87eceb369ea22d5cd73866499851cb2cca3/src/transformers/pipelines/audio_utils.py#L8).

Reply:
Thanks! If they are necessary to pass the newly added audio and vision tests, then please ignore this comment.